### ISSUE TITLE
Undefined index with upgrade.php (develop)

### ISSUE THREAD

**Issue by stefaweb** [mailwatch/MailWatch#795]
With last develop version, we got this.

```
 - optional MAILWATCH_SMTP_HOSTNAME .................................. WARNING
PHP Notice:  Undefined index: description in /home/xxxx/mailwatch-1.2.3-dev/upgrade.php on line 721

Notice: Undefined index: description in /home/xxxx/mailwatch-1.2.3-dev/upgrade.php on line 721

*** ERROR/WARNING SUMMARY ***
MailScanner.conf: HighScoringSpamActions != store (=delete)
conf.php: optional configuration entry "RPC_PORT" is missing, needed if RPC_ONLY mode is enabled
conf.php: optional configuration entry "RPC_SSL" is missing, needed if RPC_ONLY mode is enabled
conf.php: optional configuration entry "RPC_REMOTE_SERVER" is missing, needed to show number of mails in postfix queues on remote server (RPC)
conf.php: optional configuration entry "LDAP_BIND_PREFIX" is missing, needed when using LDAP authentication
conf.php: optional configuration entry "LDAP_BIND_SUFFIX" is missing, needed when using LDAP authentication
conf.php: optional configuration entry "PWD_RESET_FROM_NAME" is missing, needed if Password Reset feature is enabled
conf.php: optional configuration entry "PWD_RESET_FROM_ADDRESS" is missing, needed if Password Reset feature is enabled
conf.php: optional configuration entry "SESSION_NAME" is missing, needed if experiencing session conflicts
conf.php: optional configuration entry "MAILWATCH_SMTP_HOSTNAME" is missing,
```

**Comment by asuweb** [comment]
fixed in #796

**Comment by stefaweb** [comment]
Ok
